components/Form.js

```diff
@@ -0,0 +1,45 @@
+import { useId } from "react";
+import styled from "styled-components";
+
+export default function Form({ onSubmit, name, placeholder }) {
+  const id = useId();
+  return (
+    <StyledForm onSubmit={onSubmit}>
+      <label htmlFor={id}></label>
+      <Input
+        id={id}
+        type="text"
+        name={name}
+        placeholder={placeholder}
```

Comment:
I know it looks way more cool and clean, but for accessibility reasons, you should give your `label` a text instead of using a `placeholder`